### ISSUE TITLE
Support for Magento 2.3.x

### ISSUE THREAD

**Issue by webspeaks** [skynixukraine/Magento-2---MySQL-Cluster#3]
Hi, are there any plans to provide support for Magento 2.3.x?

Thanks

**Comment by webspeaks** [comment]
@fballiano No, I could not find any other solution. Please let me know if you have any other possible solution.

**Comment by hmphu** [comment]
@webspeaks [This fork](https://github.com/MyVitamins/Magento-2---MySQL-Cluster/commits/master) added a commit to support 2.3. I did some test with my local site 2.3.2 and it works pretty well.
Please have a try
